m4/ax_python_devel.m4: do not check for distutils

With py 3.10 this prints a deprecation warning which is
taken as an error. Upstream should rework the code to not
use distuils.

Upstream-Status: Inappropriate [needs a proper fix upstream]
Signed-off-by: Armin Kuster <akuster808@gmail.com>

Index: git/libraries/libapparmor/m4/ac_python_devel.m4
===================================================================
--- git.orig/libraries/libapparmor/m4/ac_python_devel.m4
+++ git/libraries/libapparmor/m4/ac_python_devel.m4
@@ -66,21 +66,6 @@ variable to configure. See ``configure -
         fi
 
         #
-        # Check if you have distutils, else fail
-        #
-        AC_MSG_CHECKING([for the distutils Python package])
-        ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
-        if test -z "$ac_distutils_result"; then
-                AC_MSG_RESULT([yes])
-        else
-                AC_MSG_RESULT([no])
-                AC_MSG_ERROR([cannot import Python module "distutils".
-Please check your Python installation. The error was:
-$ac_distutils_result])
-                PYTHON_VERSION=""
-        fi
-
-        #
         # Check for Python include path
         #
         AC_MSG_CHECKING([for Python include path])
